Sources/Fuzzilli/FuzzIL/AbstractInterpreter.swift

```diff
@@ -399,6 +399,28 @@ public struct AbstractInterpreter {
                 set(instr.output, .boolean)
             }
 
+        case let op as AssignmentOperation:
+            switch op.op {
+            case .Add:
+                set(instr.input(0), .primitive)
```

Comment:
I made some changes to the AbstractInterpreter's handling of bigints in https://github.com/googleprojectzero/fuzzilli/commit/7dbaa342b2fe00da8f33c8d96c7b5eb460e24fdc please copy the handling of these from e.g. the BinaryOperation again :)